test/cluster/mzworkflows.py

```diff
@@ -0,0 +1,30 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+from materialize.mzcompose import Dataflowd, Materialized, Testdrive, Workflow
+
+daemons = [
+    Dataflowd(name="dataflowd", options="--workers 1", hostname="dataflowd"),
+    Materialized(
+        options="--workers 1 --dataflowd-addr dataflowd:6876",
+        mzbuild="coordd",
+        depends_on=["dataflowd"],
+    ),
+]
+
+services = [*daemons, Testdrive()]
+
+
+def workflow_cluster(w: Workflow):
+    w.start_and_wait_for_tcp(services=daemons, timeout_secs=5)
```

Comment:
This timeout may be too low for our CI. What required changing this from the default for 30 seconds?